components/802.15.4_RF/stm-s2lp-rf-driver/source/NanostackRfPhys2lp.cpp

```diff
@@ -572,6 +575,46 @@ static void rf_set_channel_configuration_registers(void)
     rf_write_register(PCKTCTRL5, preamble_len);
 }
 
+#if MBED_CONF_S2LP_EXTERNAL_FRONTEND
+static void rf_configure_external_frontend(PinName ant_sel)
+{
+    if (ant_sel != NC) {
+        DigitalOut ant_sel_out(ant_sel);
+        ant_sel_out = MBED_CONF_S2LP_ANTENNA_SELECTION;
+    }
+
+    rf_write_register_field(S2LP_CSD_PIN, GPIO_FUNCTION_FIELD, TX_RX_INDICATION);
+    rf_write_register_field(S2LP_CSD_PIN, GPIO_MODE_FIELD, GPIO_DIGITAL_OUT_LOW);
+
+    rf_write_register_field(S2LP_CPS_PIN, GPIO_FUNCTION_FIELD, RX_INDICATION);
+    rf_write_register_field(S2LP_CPS_PIN, GPIO_MODE_FIELD, GPIO_DIGITAL_OUT_LOW);
+
+    rf_write_register_field(S2LP_CTX_PIN, GPIO_FUNCTION_FIELD, TX_STATE_INDICATION);
+    rf_write_register_field(S2LP_CTX_PIN, GPIO_MODE_FIELD, GPIO_DIGITAL_OUT_LOW);
+
+    tx_power_dbm = 10;
```

Comment:
Should the default power level be something more conservative example 0 dbm

Reply:
Okey will change